HandleNHCreateOrUpdateInstallationCallActivity/handler.ts

```diff
@@ -14,7 +14,7 @@ import { createOrUpdateInstallation } from "../utils/notification";
 import { NotificationHubConfig } from "../utils/notificationhubServicePartition";
 
 // Activity input
-export type ActivityInput = t.TypeOf<typeof ActivityInput>;
+export type CreateOrUpdateActivityInput = t.TypeOf<typeof ActivityInput>;
```

Comment:
@balanza why the need of renaming it, and then exporting it as `ActivityInput` in `index.ts`?

Reply:
Good catch, it was my IDE that messed that. Fixed in https://github.com/pagopa/io-functions-pushnotifications/pull/20/commits/7921ca44dca5c2d9c4f1a6b477bbc4536b6ca85a